### DISCUSSION TITLE
Question, when switching checkpoints

### DISCUSSION THREAD

**Discussion by P-A-Ori** [DominikDoom/a1111-sd-webui-tagcomplete#150] [Q&A]
Hi there,

I really enjoy using this extension. I have a question about its functionality. Whenever I switch checkpoints, I've noticed that the extension doesn't work unless I restart the UI. Is this a problem with my system, or is it a known issue with the extension?

Also, this is my first time asking a question on Github. If you need any additional information from me, please let me know. Please excuse any errors in my English, as I am using a translation site to communicate.

Thank you.

**Comment by DominikDoom** [comment]
It seems to be a problem some people have at the moment, although it doesn't happen to me.
I'm closing this to continue it in #149, which will be easier for me to follow. Please also try what I've written there and disable other extensions, to see if the issue also happens standalone.
